### DISCUSSION TITLE
0.3.4

### DISCUSSION THREAD

**Discussion by firstof9** [moralmunky/Home-Assistant-Mail-And-Packages#666] [Releases]
[![Downloads for this release](https://img.shields.io/github/downloads/moralmunky/Home-Assistant-Mail-And-Packages/0.3.4/total.svg)](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/releases/0.3.4)

## :boom: Breaking Change :boom:

- fix: change to extra_state_attributes @firstof9 (#600)
- 0.3.4 - Release @firstof9 (#648)

## :zap: Enhancements :zap:

- feat: make mail_updated a timestamp class @firstof9 (#614)

## :sparkles: New Features :sparkles:

- feat: make amazon sensor more configurable @firstof9 (#608)
- feat: Add support for Polish shippers + devcontainer @kubawolanin (#620)
- feat: add diagnostics @firstof9 (#632)

## :bug: Bug Fixes :bug:

- fix: update fedex subject search @firstof9 (#588)
- fix: add fedex delivered subject @firstof9 (#590)
- fix: update UPS delivering subject @firstof9 (#593)
- fix: amazon handle arriving early emails @firstof9 (#598)
- fix: strip `>` from forwarded amazon date capture @firstof9 (#619)
- fix(shipper): add amazon.com.au and update auspost_delivering @kaizersoje (#625)
- fix: resolve utf-8 subject searching error @firstof9 (#630)
- fix: remove utf-8 encoding on default imap search @firstof9 (#631)
- fix(amazon): add german language support @firstof9 (#634)
- fix: changes for HA core 2022.4 @firstof9 (#644)
- fix: spelling mistake in UI @firstof9 (#647)
- fix(shippers): Add extra matcher for USPS Informed Delivery @keslerm (#656)

## :mortar_board: Code Quality :mortar_board:

- refactor: adjust const import @firstof9 (#597)
- test: use tox for testing @firstof9 (#626)
- refactor: make amazon easier to manage @firstof9 (#627)
- fix: update translations @firstof9 (#657)
- fix: remove quotes from SINCE date @firstof9 (#663)

## :books: Documentation :books:

- docs: update hacs info.md @firstof9 (#645)

## Links
- [Wiki](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki)
- [Submit bugs/feature requests](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/releases/tag/0.3.4'>0.3.4</a>.</em>

**Comment by GizmoGoody** [comment]
Thank you.

I went to check out the new configuration options (e.g. Amazon days), but don't see the new field. Do I have to delete and reinstall?

**Reply by firstof9** [reply]
Very strange I don't know what's going on there. Must be some weirdness from Github, I'll try a re-release.

**Reply by firstof9** [reply]
`0.3.4-2` seems to have worked. Sorry about that.

**Reply by GizmoGoody** [reply]
Hmmm. I'm not seeing a prompt for another update under Integrations.

**Reply by firstof9** [reply]
In HACS click the overflow menu (the 3 dots) and hit "Redownload", from there you'll be able to see it.

**Reply by GizmoGoody** [reply]
Thanks, it’s working again now. I have noticed after a few times trying to reconfigure on -1 and now -2 that I always have to re-enter the gmail address to get Step 2 to finish.
![A2EA6A92-1BC2-4D47-BA66-812E43B11494](https://user-images.githubusercontent.com/46235745/169432108-882fcc3a-fbac-4c56-ad46-bda029c801e7.jpeg)

**Reply by firstof9** [reply]
You can backspace 1 character and retype it, that will also do it.
It's a limitation of the Home Assistant configuration UI unfortunately.
